fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -799,9 +799,19 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static boolean enable_strict_storage_medium_check = false;
 
+    /**
+     * If set to true, FE will choose backends on the same host which can be used for better dev because
+     * some cases can only be reproduced on the distributed env.
+     * <p>
+     * NOTE: This config should not be used on the production because data's replicas on the
+     * same host may cause data lost when the host is broken down.
+     */
+    @ConfField(mutable = false)
+    public static boolean enable_replicas_on_the_same_host = false;
```

Comment:
public static enable_pseudo_distributed_mode;
It is only used in test mode. Not recommended in the production.